### PR TITLE
comment out unused import of com.ibm.rmi.corba

### PR DESCRIPTION
**Pull request by ejpring**

### CODE REVIEW

File: com.ibm.streamsx.messaging/impl/java/src/com/ibm/streamsx/messaging/jms/EmptyMessageHandler.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 import javax.jms.StreamMessage;
 import javax.jms.TextMessage;
 
-import com.ibm.rmi.corba.ObjectManager;
+//import com.ibm.rmi.corba.ObjectManager; ... not used
```
Comment (ddebrunner) [inline com.ibm.streamsx.messaging/impl/java/src/com/ibm/streamsx/messaging/jms/EmptyMessageHandler.java]: Why not just remove it?

This is under source control, so any older code is always available.

Comment (ejpring) [inline com.ibm.streamsx.messaging/impl/java/src/com/ibm/streamsx/messaging/jms/EmptyMessageHandler.java]: That would be fine with me, too, but I wasn't sure what 'EmptyMessageHandler' means in this context. If its a skeleton to be filled out later, then that import might be needed by code added in the future. But if its really just what's coded now, deleting the import statement is probably the right thing to do.

Comment (chanskw) [inline com.ibm.streamsx.messaging/impl/java/src/com/ibm/streamsx/messaging/jms/EmptyMessageHandler.java]: EmptyMessageHandler is used as a "Test" message handler.
There is a special kind of message in JMS where the message payload is empty.
I think that's when this EmptyMessageHandler is used.

Comment (conglisc) [inline com.ibm.streamsx.messaging/impl/java/src/com/ibm/streamsx/messaging/jms/EmptyMessageHandler.java]: I believe this import statement is safe to remove, I am guessing the original author might be exploring different approaches with object via this import statement

**Comment by chanskw** [comment]
Thanks Ed.  I am merging this in.

**Pull request merged**